Sources/SwiftSyntax/generated/syntaxNodes/SyntaxNodesTUVWXYZ.swift

```diff
@@ -2588,6 +2588,98 @@ public struct TypeInitializerClauseSyntax: SyntaxProtocol, SyntaxHashable, _Leaf
   }
 }
 
+// MARK: - TypeSpecifierSyntax
+
+/// A specifier that can be attached to a type to eg. mark a parameter as `inout` or `consuming`
+///
+/// ### Children
+/// 
+///  - `specifier`: (`inout` | `__shared` | `__owned` | `isolated` | `_const` | `borrowing` | `consuming` | `transferring` | `_resultDependsOn`)
+///
+/// ### Contained in
+/// 
+///  - ``TypeSpecifierListSyntax``
+public struct TypeSpecifierSyntax: SyntaxProtocol, SyntaxHashable, _LeafSyntaxNodeProtocol {
+  public let _syntaxNode: Syntax
+  
+  public init?(_ node: some SyntaxProtocol) {
+    guard node.raw.kind == .typeSpecifier else {
+      return nil
+    }
+    self._syntaxNode = node._syntaxNode
+  }
+  
+  /// - Parameters:
+  ///   - leadingTrivia: Trivia to be prepended to the leading trivia of the node’s first token. If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
+  ///   - trailingTrivia: Trivia to be appended to the trailing trivia of the node’s last token. If the node is empty, there is no token to attach the trivia to and the parameter is ignored.
```

Comment:
Miss `specifier` here? And `specifiers` above actually.

Reply:
Generation-wise that’s intentional because `specifier` doesn’t have documentation. But I should probably write some…